### PR TITLE
chore(#25): Update README

### PR DESCRIPTION
**Pull request by andrablaj**
# Description

Update and clarify existing README. Add the sequence diagram created by @Phil-Mwago, with version control support as suggested by @mrjones-plip.

#25 

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.
 
# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by andrablaj** [comment]
@njogz @samuelimoisili when using Docker setup, I am not sure if:

> Create a file named `.env` copy over the contents of `.env.template` and update the `CHT_USERNAME` and `CHT_PASSWORD` values with the admin credentials of your CHT instance.

is still needed. Can you please help clarify that bit?

File: README.md [hunk 1]
```diff
@@ -67,7 +68,6 @@ The following steps apply when running CHT locally in development mode and when
 
 #### CHT Development Environment
 1. Set up a local CHT instance using [these instructions](https://docs.communityhealthtoolkit.org/apps/tutorials/local-setup/).
```
Comment (lorerod) [inline README.md:70]: Following [these instructions](https://docs.communityhealthtoolkit.org/apps/tutorials/local-setup/) I get to a port conflict with interoperability. More detail on issue [comment](https://github.com/medic/interoperability/issues/24#issuecomment-1468453919)

**Review by lorerod — CHANGES_REQUESTED** [review]
Thank you @andrablaj for this! . I added a couple of suggestions that may be helpful. Let me know what you think.

File: README.md [hunk 2]
```diff
@@ -5,40 +5,90 @@ This project implements a Loss to Follow Up (LTFU) workflow system for CHIS base
...
-1. Create a new user in the CHT instance with the username `interop-client` using these [instructions](https://docs.communityhealthtoolkit.org/apps/tutorials/contact-and-users-1/#4-create-the-chw-user). For the role you can select `Data entry` and `Analytics` roles. Please note that you can use any username you prefer but you would have to update the the config with the new username. You can do that by editing the `cht-config/app_settings.json` file and updating the `username` value in the `outbound` object e.g. on this [line](https://github.com/medic/interoperability/blob/main/cht-config/app_settings.json#L452).
-1. Securely save the `interop-client` user's password to the database using the instructions [here](https://docs.communityhealthtoolkit.org/apps/reference/api/#credentials). Change the values `mykey` and `my pass` to `openhim1` and your user's password respectively. An example of the curls request is below:
+If everything is successful you should see this:
+ ![](./docs/images/good-client-screen.png)
+
+### CHT configuration with Docker
+The following steps apply when running CHT via the Docker setup provided in this repository:
+
```
[the remark this answers is not in the document]
Comment (andrablaj) [reply README.md:57]: They don't need to run the container as they already did via` ./startup init.` These steps are only to check the CHT configuration. If you have suggestions on how to make it clearer, please comment here.

**Comment by andrablaj** [comment]
@lorerod you requested to add the OpenHIM instance credentials in the `README`. We might not want to share them publicly, so I will not add them for now. This can be discussed in the next weekly call. 
FYI @njogz

File: README.md [hunk 3]
```diff
@@ -8,11 +8,13 @@ A first version of the project can be found in the [chis-interoperability](https
 ### Services
 Services are currently available at these URLs:
 
-* **OpenHIM Admin Console** - TODO
+* **OpenHIM Admin Console** - [https://interoperability.dev.medicmobile.org](https://interoperability.dev.medicmobile.org)
```
[the remark this answers is not in the document]
Comment (andrablaj) [reply README.md]: Not sure if we want to share the instance credentials publicly. To be discussed in a next call with the team.

**Pull request merged**